examples/CVAE_example/README.md

```diff
@@ -0,0 +1,53 @@
+# Federated Conditional Variational Auto-encoder Example
+This is an example implementation of training a Conditional Variational auto-encoder model on the MNIST dataset. In this example, the model is conditioned based on a client-specific ID that is an integer associated with the client. This lets the model learn a specific mapping of the input to the latent representation while being conditioned on the specific client feature space, potentially leading to more personalized representations. In this example, the client is a BasicClient, but it can instead inherit from any of the available client classes. The server uses Federated Averaging to aggregate the CVAE model weights.
+
+## Running the Example
```

Comment:
Small note: In the most recently merge PR, Marcelo updated the instructions for installation in the examples to reference the README, you probably just copy and past that to replace these old installation instructions